src/output/dump_metrics.rs

```diff
@@ -0,0 +1,258 @@
+use std::io::Write;
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, StandardStreamLock, WriteColor};
+
+use crate::cyclomatic;
+use crate::exit;
+use crate::fn_args;
+use crate::halstead;
+use crate::loc;
+use crate::mi;
+use crate::nom;
+
+use crate::spaces::{CodeMetrics, FuncSpace};
+
+pub(crate) fn dump_root(space: &FuncSpace) -> std::io::Result<()> {
```

Comment:
is it still possible to dump using this stuff ?

Reply:
I can't understand the question. Could you elaborate more on this one please?

Reply:
I tried the patch locally and it's ok.
My question was just to know if it's still possible to dump metrics on stdout in using the code here. 
And using single --metrics is ok.

Reply:
Oh, I see. I'm glad you solved it :)